fabric-fluid-v1/src/main/java/net/fabricmc/fabric/impl/fluids/BucketItemAccessor.java

```diff
@@ -0,0 +1,7 @@
+package net.fabricmc.fabric.impl.fluids;
+
+import net.minecraft.fluid.Fluid;
+
+public interface BucketItemAccessor {
+	Fluid getFluid();
```

Comment:
should change to an accessor

Reply:
there's already a BucketItemMixin that has internal logic, no point in doing that

Reply:
just wanted to use accessor so you don't get conflicts right away if another dumb mod does that